internal/commands/pull/command.go

```diff
@@ -60,32 +61,58 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
+	clientZipPkgs, err := cmd.inputs.resolveClientTemplates(ui, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	projectPath, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	proceed, err := checkAppDestination(ui, pathTarget)
+	var pathFrontend string
+	pathBackend := projectPath
+	if len(clientZipPkgs) != 0 {
+		pathFrontend = filepath.Join(projectPath, local.FrontendPath)
+		if proceed, err := checkPathDestination(ui, pathFrontend); err != nil {
+			return err
+		} else if !proceed {
+			return nil
+		}
+		pathBackend = filepath.Join(projectPath, local.BackendPath)
+	}
+
+	// App path
+	proceed, err := checkPathDestination(ui, pathBackend)
 	if err != nil {
 		return err
 	} else if !proceed {
 		return nil
 	}
 
+	pathRelative, err := filepath.Rel(profile.WorkingDirectory, projectPath)
+	if err != nil {
+		return err
+	}
+
 	if cmd.inputs.DryRun {
-		ui.Print(
-			terminal.NewTextLog("No changes were written to your file system"),
-			terminal.NewDebugLog("Contents would have been written to: %s", pathRelative),
-		)
+		if len(cmd.inputs.TemplateID) != 0 {
+			ui.Print(
+				terminal.NewTextLog("No changes were written to your file system"),
+				terminal.NewDebugLog("App contents would have been written to: %s", filepath.Join(pathRelative, local.BackendPath)),
+				terminal.NewDebugLog("Template contents would have been written to: %s", filepath.Join(pathRelative, local.FrontendPath)),
+			)
+		} else {
+			ui.Print(
+				terminal.NewTextLog("No changes were written to your file system"),
+				terminal.NewDebugLog("Contents would have been written to: %s", pathRelative),
+			)
+		}
 		return nil
 	}
 
-	if err := local.WriteZip(pathTarget, zipPkg); err != nil {
+	if err := local.WriteZip(pathBackend, zipPkg); err != nil {
```

Comment:
if we wanted, right before this would be where I'd add `checkPathDestination` for the `pathBackend` filepath